workshops/painting_app/painting_app.md

```diff
@@ -0,0 +1,323 @@
+---
+name: Painting app
+description: Create a painting app for your website
+author: '@wollygfx'
+img: ''
+---
+
+In this workshop you'll learn how to make a painting application using HTML, CSS & JavaScript, follow along with me and see how easy it is.
+
+Here you can find a [live demo](https://css.wollygfx.repl.co) and here, you will find the [source code](https://repl.it/@wollygfx/css#index.html).
+
+
+## Set Up
+
+This workshop requires a very basic knowledge of the following languages: HTML & JS. Don’t worry if you get stuck at some point in the workshop, everything is explained the best way for you to understand!
+
+For this workshop we will use [Repl.it](https://repl.it), click [here](https://repl.it/languages/html) to create a coding environment right for this workshop.
+
+![Setup](https://cloud-qbmylslty.vercel.app/0image.png)
+
+## HTML
+
+In this part of the workshop will go all the elements that are part of the application, which are: buttons, inputs, and a canvas.
+
+So let's start by creating a `div` container with the class `main-container` inside of our `<body>` tag. In this container we'll put all of our html elements.
+```html
+<div class="main-container"></div>
+```
+
+Then, we are creating another `div` container, that will hold all the color button. Inside of this `div` container, we'll put 6 `button` elements, with the following values:
+```html
+<div class="main-container">
+	<div class="colors">
+		<button value="#FBFF00"></button>
+		<button value="#009fff"></button>
+		<button value="#000000"></button>
+		<button value="#FF0000"></button>
+		<button value="#3EFF00"></button>
+		<button value="#FFF"></button>
+	</div>
+</div>
+```
+Each button represent its respective HEX code.
+
+Finally, right after the `div` container with the class `colors`, we'll put 4 elements with the following values:
+```
+<div class="main-container">
+	<div class="colors">
+		<button value="#FBFF00"></button>
+		<button value="#009fff"></button>
+		<button value="#000000"></button>
+		<button value="#FF0000"></button>
+		<button value="#3EFF00"></button>
+		<button value="#FFF"></button>
+	</div>
+
+   <--! Here we put the 4 elements-->
+
+	<input id="brush" type="range" value="1" min="1" max="5" step="1"></input>
+	<button id="clear" type="button">Clear</button>
+	<button id="save" type="button">Save</button>
+	<br>
+	<canvas id="paint-canvas" width="640" height="400"></canvas>
+
+</div>
+```
+Let's break this down:
+- The `input` element is used to create interactive controls for web-based forms in order to accept data from the user, since the type of input we are using is `range`, it requires you to set the attributes `value`, `min`, `max` and `step` which basically set the initial value for the input, the max and min value for it, and the step from a value to other respectively
```

Comment:
```suggestion
- The `input` element is used to create interactive controls for web-based forms in order to accept data from the user, since the type of input we are using is `range`, it requires you to set the attributes `value`, `min`, `max` and `step` which basically sets the initial value for the input, the max and min value for it, and the step from a value to other respectively.
```